### ISSUE TITLE
find_center_vo raises TypeError: 'numpy.float64' object cannot be interpreted as an index

### ISSUE THREAD

**Issue by DimitarTasev** [tomopy/tomopy#252]
Hi, I'm trying to use `find_center_vo` to find the center of rotation on a sample, and compare the results to other methods. However only the `find_center_vo` raises a `TypeError: 'numpy.float64' object cannot be interpreted as an index`, the full stack trace:

```
Traceback (most recent call last):
  File "main.py", line 89, in <module>
    main()
  File "main.py", line 72, in main
    res = runner.execute(config)
  File "/home/matt/Documents/imgpy/imopr/runner.py", line 28, in execute
    return action(sample, flat, dark, config, indices)
  File "/home/matt/Documents/imgpy/imopr/corvo.py", line 39, in execute
    drop=20)
  File "/home/matt/Documents/imgpy/recon/tools/tomopy_tool.py", line 28, in find_center_vo
    return self._tomopy.find_center_vo(**kwargs)
  File "/home/matt/anaconda2/lib/python2.7/site-packages/tomopy/recon/rotation.py", line 251, in find_center_vo
    init_cen = _search_coarse(_tomo, smin, smax, ratio, drop)
  File "/home/matt/anaconda2/lib/python2.7/site-packages/tomopy/recon/rotation.py", line 277, in _search_coarse
    mask = _create_mask(2 * Nrow - 1, Ncol, 0.5 * ratio * Ncol, drop)
  File "/home/matt/anaconda2/lib/python2.7/site-packages/tomopy/recon/rotation.py", line 338, in _create_mask
    mask[i, p1:p2 + 1] = np.ones(p2 - p1 + 1, dtype='float32')
  File "/home/matt/.local/lib/python2.7/site-packages/numpy/core/numeric.py", line 192, in ones
    a = empty(shape, dtype, order)
TypeError: 'numpy.float64' object cannot be interpreted as an index
```

As far as I've seen, the error happens in `_create_mask` in `rotation.py`, because `p1` and `p2` are floats when used in the array indices. 

The Version of numpy is 1.12.0, and the reason for this error is that using floats for indices are removed in 1.12

**Comment by DimitarTasev** [comment]
A temporary solution (from https://github.com/rbgirshick/py-faster-rcnn/issues/481) is to revert numpy to 1.11 with `sudo pip install -U numpy==1.11.0` if anyone else gets this error

**Comment by dgursoy** [comment]
Thanks @DimitarTasev. I'll keep this open till this is fixed.

**Comment by nghia-vo** [comment]
Thanks. It has been fixed #255
